build/nuget/uno.winui.common.targets

```diff
@@ -221,12 +221,8 @@
 	<Target Name="_ValidateIOSStaticRegistrar"
 			BeforeTargets="BeforeBuild"
 			Condition="
-			'$(UnoDisableValidateIOSStaticRegistrar)'==''
-			and (
-				('$(TargetFramework)'=='net6.0-ios' and '$(RuntimeIdentifier)'=='iossimulator-x64')
-				or ('$(ProjectTypeGuids)'=='{FEACFBD2-3405-455C-9665-78FE426C6842};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}' and '$(Platform)'=='iPhoneSimulator')
```

Comment:
I think the `ProjectTypeGuids` check corresponds to Xamarin projects where are no longer supported.